src/main/resources/catalog.json

```diff
@@ -0,0 +1,181 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache license, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the license for the specific language governing permissions and
+ * limitations under the license.
+ */
+ {
+  "products" : [ ],
+  "categories" : [ ],
+  "events" : [ {
+    "id" : 1,
+    "name" : "login",
+    "displayName" : "Login",
+    "description" : "User Login",
+    "aliases" : [ ],
+    "attributes" : [ {
+      "name" : "timestamp",
+      "required" : true
+    }, {
+      "name" : "userId",
+      "required" : true
+    }, {
+      "name" : "requestId",
+      "required" : false
```

Comment:
I'm thinking we should probably require a requestId on all events. We'll generate a new request id in a servlet filter and in the CLI.